docs/source/pages/using.rst

```diff
@@ -110,6 +110,24 @@ With the example above, the generated output reports can be seen here:
 - `micronaut-core.html <../_static/examples/micronaut-projects/micronaut-core/analyze_with_sbom/micronaut-core.html>`__
 - `micronaut-core.json <../_static/examples/micronaut-projects/micronaut-core/analyze_with_sbom/micronaut-core.json>`__
 
+'''''''''''''''''''''''''''
+Analyzing more dependencies
+'''''''''''''''''''''''''''
+
+In some cases the dependencies that Macaron discovers lack a direct connection to a repository for it to analyze. To improve results in these instances, the Repository Finding feature can be enabled. This feature makes use of a dependency's identifying information that can be found using the package registries located on the Internet.
+
+.. note:: The Repository Finding feature currently only works for Java projects via SCM meta data found within artifact POM files.
+
+This feature is enabled by default. To disable, or configure its behaviour in other ways, a custom ``defaults.ini`` should be passed to Macaron during execution. Under the ``repofinder.java`` header, five options exist: ``find_repos``, ``artifact_repositories``, ``repo_pom_paths``, ``find_parents``, ``artifact_ignore_list``. These options behave as follows:
```

Comment:
It would be helpful to add a reference like this:

```
See :ref:`dump-defaults <reference to docs/source/pages/cli_usage/action_dump_defaults.rst >`, the CLI command to dump the default configurations in ``defaults.ini``. After making changes, see :ref:`analyze <analyze-action-cli>` CLI command for the option to pass the modified ``defaults.ini`` file.
```

You need to add a reference in `docs/source/pages/cli_usage/action_dump_defaults.rst` but `analyze-action-cli` reference already exists.

Reply:
I've tried to. Not sure if it's displaying as it should be though.